pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -185,33 +185,37 @@ func (e *eksOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 		logrus.Infof("waiting for cluster EKS [%s] create failure to be resolved", cluster.Name)
 		return e.setFalse(cluster, apimgmtv3.ClusterConditionProvisioned, failureMessage)
 	case "active":
-		if apimgmtv3.ClusterConditionPending.IsUnknown(cluster) {
```

Comment:
Do we not need this section anymore for rancher-created clusters ?

Reply:
Rancher created ones go: Provisioning -> waiting -> active
Imported: pending -> waiting -> active